src/main/scala/com/fengjian/blog/common/LocalDateTimeFormatterUtils.scala

```diff
@@ -0,0 +1,14 @@
+package com.fengjian.blog.common
+
+import io.circe.Encoder
+
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+object LocalDateTimeFormatterUtils {
+
+  private val fmt = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")
+
+  implicit val dateEncoder: Encoder[LocalDateTime] = Encoder.encodeString.contramap[LocalDateTime](_.format(fmt))
```

Comment:
If you want to define a util here, it's not a good practice to make it a `implicit` encoder.